aruba/features/renew/renew-with-csr-service.feature

```diff
@@ -59,7 +60,10 @@ Feature: renew action with `-csr service` option
         And it should write certificate to the file "c1.pem"
         And private key in "k1.pem" and certificate in "c1.pem" should have the same modulus
 
-  @TODO # This scenario is not currently working due to reuse csr not being implemented yet in OutagePredict
+  @TODO @VAAS # This scenario is not currently working due to reuse csr not being implemented yet in OutagePredict
```

Comment:
If there is an update here, remove the previos comment as it no longer applies. It only makes explanation more complex than required.

Reply:
I mostly may need to edit it, no longer applies, but requires verification (hence not removed the "TODO"). Will clarify in next commit

Reply:
Done by latest changes